pegasus/sites.v3/code.org/views/teach_page_labs.haml

```diff
@@ -0,0 +1,53 @@
+%h2
+  =hoc_s(:"teach_labs_section.header")
+%p
+  =hoc_s(:"teach_labs_section.description")
+.action-block__wrapper.action-block__wrapper--three-col
```

Comment:
Nit: I usually do a loop for these. It makes it easier to update as the list grows, or if one of them needs to be updated.

Reply:
I initially did that but then found it easier to do it this way I think. I'm trying to remember my exact motivation...

Reply:
I think it was just the same amount of complexity imo as every line is customized but I thought there was something else. But it's been a week and I can't remember

Reply:
They all have the same elements so you could do something similar to [this](https://github.com/code-dot-org/code-dot-org/blob/5ad1e6d9036b86913713676b89b2c019126f3764/pegasus/sites.v3/code.org/views/game_design/game_design_curriculum_mid_high.haml#L35-L48), but I don't think it's worth updating at this point.